content/en/docs/setup/additional-setup/gateway/index.md

```diff
@@ -0,0 +1,296 @@
+    matchLabels:
+      istio: ingressgateway
+  template:
+    metadata:
+      annotations:
+        # Enable gateway injection
+        inject.istio.io/templates: gateway
+      labels:
+        # Set a unique label for the gateway. This is required to ensure Gateways
+        # can select this workload
+        istio: ingressgateway
+        # Enable gateway injection. If connecting to a revisioned control plane, replace with
+        # istio.io/rev: revision-name
+        sidecar.istio.io/inject: "true"
+    spec:
+      containers:
+      - name: istio-proxy
+        image: auto # The image will automatically update each time the pod starts.
+---
+# Set up roles to allow reading credentials for TLS
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: istio-ingressgateway-sds
+  namespace: istio-ingress
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: istio-ingressgateway-sds
+  namespace: istio-ingress
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: istio-ingressgateway-sds
+subjects:
+- kind: ServiceAccount
+  name: default
+{{< /text >}}
+
+Next, apply it to the cluster:
+
+{{< text bash >}}
+$ kubectl create namespace istio-ingress
+$ kubectl apply -f ingress.yaml
+{{< /text >}}
+
+{{< tip >}}
+The example above enables injection on the pod. This can also be controlled at the namespace level.
+See [Controlling the injection policy](/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy) for more information.
+{{< /tip >}}
+
+{{< /tab >}}
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+A call to `istioctl install` with [default settings](/docs/setup/install/istioctl/#install-istio-using-the-default-profile) will deploy a gateway by default.
+However, this couples it to the control plane, making management and upgrade more complicated.
+It is highly recommended to decouple these and allow independent operation.
+This can be done by selecting the `minimal` profile, for example `istioctl install --set profile=minimal`.
+
+First, setup an `IstioOperator` configuration file, called `ingress.yaml` here:
+
+{{< text yaml >}}
+apiVersion: operator.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  name: ingress
+spec:
+  profile: empty # Do not install CRDs or the control plane
+  components:
+    ingressGateways:
+    - name: ingressgateway
+      namespace: istio-ingress
+      enabled: true
+      label:
+        # Set a unique label for the gateway. This is required to ensure Gateways
+        # can select this workload
+        istio: ingressgateway
+  values:
+    gateways:
+      istio-ingressgateway:
+        # Enable gateway injection
+        injectionTemplate: gateway
+{{< /text >}}
+
+Then install using standard `istioctl` commands:
+
+{{< text bash >}}
+$ kubectl create namespace istio-ingress
+$ istioctl install -f ingress.yaml
```

Comment:
I assume the operator and helm approach doesn't require label the namespace Istio-ingress with auto sidecar injection?

Reply:
None of the methods do. All of them explicitly turn injection on in the deployment. This is following standard injection logic - https://preliminary.istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy. Nothing here is special for gateways.

If they don't want to label the deployment, they can just label the namespace.